src/themes/default/elements/icon.overrides

```diff
@@ -201,6 +202,19 @@ i.icon.other.gender.horizontal:before { content: "\f22b"; }
 i.icon.neuter:before { content: "\f22c"; }
 i.icon.genderless:before { content: "\f22d"; }
 
+/* Accessibility */
+i.icon.universal.access:before { content: "\f29a"; }
+i.icon.alternative.wheelchair:before { content: "\f29b"; }
+i.icon.blind:before { content: "\f29d"; }
+i.icon.audio.description:before { content: "\f29e"; }
+i.icon.volume.control.phone:before { content: "\f2a0"; }
+i.icon.braille:before { content: "\f2a1"; }
+i.icon.assistive.listening.systems:before { content: "\f2a2"; }
+i.icon.american.sign.language.interpreting:before { content: "\f2a3"; }
```

Comment:
New icon: `american sign language interpreting`, the American standard of sign interpreting. Depicts a pair of hands doing what it seems to be the word "INTERPRET".